### PR TITLE
Bug 1093569 — Add notifications on participant leave

### PR DESCRIPTION
**Pull request by ametaireau**
https://bugzilla.mozilla.org/show_bug.cgi?id=1093569

### CODE REVIEW

**Comment by ametaireau** [comment]
So,here is a way to have notifications from Redis. Travis-CI currently doesn't seem to support redis notifications, so I've opened https://github.com/travis-ci/travis-ci/issues/3105.

Maybe we should skip it in this environment for now.

File: loop/notifications.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+    self.pubsub.on("pmessage", function(pattern, channel, key) {
+      Object.keys(self.observers).forEach(function(prefix) {
+        if (key.indexOf(prefix) === 0) {
+          self.observers[prefix](key);
+        }
+      })
+    });
+
+    self.pubsub.psubscribe("__keyevent*__:expired");
```
Comment (Natim) [inline loop/notifications.js]: Could we add the key pattern inside the psubscribe? i.e: ``self.pubsub.psubscribe("__keyevent__roomparticipant.*__:expired");``

File: loop/pubsub.js [hunk 2]
```diff
@@ -20,6 +20,7 @@ function RedisPubSub(options, logError) {
 
   // Let's report errors when they occur.
   client.on('error', logError);
+  client.config('set', 'notify-keyspace-events', 'Ex');
```
Comment (Natim) [inline loop/pubsub.js]: Where is the callback?

File: loop/notifications.js (hunk 1 above)
Comment (ametaireau) [inline loop/notifications.js]: nope, the events aren't conveing any key information.

File: loop/pubsub.js (hunk 2 above)
Comment (ametaireau) [inline loop/pubsub.js]: yeah, where is it!?

**Pull request merged**